include/libcgroup/config.h

```diff
@@ -29,13 +29,57 @@ extern "C" {
  * and applications can benefit from them.
  */
 
+/*
+ * cgroup systemd settings.
+ */
+struct cgroup_systemd_opts {
```

Comment:
Given that, `systemd` will be heavily used and it will be enabled by default in the configurable option, we can very well remove it, on the other hand, we would wait for the feedback from the community and remove it from the release + 1 having configurable support for systemd, after fixing any bugs reported.